README.md

```diff
@@ -62,33 +62,36 @@ Thank you to all those who have translated asbplayer:
 If you are a non-English native, and would like to help translate asbplayer, please contact me on [Discord](https://discord.gg/ad7VAQru7m).
 
 ## Getting Started
+ - *Note: asbplayer can either be used purely as a tool for subtitle control or as a deck mining tool (in which case you will need the subtitle functionality to mine).

```

Comment:
I think this line would be use case for the note syntax described here: https://github.com/orgs/community/discussions/16925

Also, I understand what you mean by "deck mining" but strictly speaking it should be "sentence mining" or "flashcard creation."